packages/api/src/submittable/createClass.ts

```diff
@@ -69,10 +69,10 @@ export default function createClass <ApiType extends ApiTypes> ({ api, apiType,
     // signs a transaction, returning `this` to allow chaining. E.g.: `sign(...).send()`
     //
     // also supports signing through external signers
-    public signAsync (account: AddressOrPair, optionsOrNonce: Partial<SignerOptions>): SubmittableThis<ApiType, this> {
+    public signAsync (account: AddressOrPair, optionsOrNonce?: Partial<SignerOptions>): SubmittableThis<ApiType, this> {
       return decorateMethod(
         (): Observable<this> =>
-          this.#observeSign(account, optionsOrNonce).pipe(mapTo(this))
+          this.#observeSign(account, {}).pipe(mapTo(this))
```

Comment:
This means we will never pass the options through. This needs to be `optionOrNonce`, as was passed. (The signature would change with default initialization)

Reply:
ok!